apps/omg_child_chain_rpc/priv/swagger/operator_api_specs/block/response_schemas.yaml

```diff
@@ -12,3 +12,5 @@ BlockResponseSchema:
         hash: '0x2733e50f526ec2fa19a22b31e8ed50f23cd1fdf94c9154ed3a7609a2f1ff981f'
         transactions:
         - '0xf8d083015ba98080808080940000...'
+        inserted_at: '2020-02-10T12:07:32Z'
```

Comment:
It seems you're not modifying the child chain rpc and I think those blocks will be served without those 2 new fields (they're still served from the RocksDB store).

(same comment from `schemas.yaml`)

Reply:
ah, yeah...duh, good point and catch. fixed thanks! ;)